### DISCUSSION TITLE
repo import failed --> cannot delete repository

### DISCUSSION THREAD

**Discussion by amamory** [community/community#10595] [Projects and Issues]
I tried to import a repository from a private gitlab to this github organization. The import process failed, I am not sure why.  But this is not the problem i want to report.  The problem is that github created a kind of `zombie` repository [here](https://github.com/fred-framework/dart_ips/). This repository cannot be deleted or anything else. Every time i click on it, it returns to the repository import page, as I would need to have a successful import to turn this zombie repo into an complete repo.

Now I have a useless repo in my organization. How can i delete this repo ?

cheers!
Alexandre

**Comment by Tpaefawzen** [comment]
I have met such the issue, too. I modified the URL to open `https://github.com/example/foo-bar/settings` to reach to the settings page successfully. Finally I deleted it.

**Reply by caseyokane-8451** [reply]
Also have this problem. Tried the URL (able to get to the settings page and the "danger zone" but still seeing "Access to this repository has been disabled while it is being migrated" when attempted to delete it). 

Have also tried doing so via the API but still no luck

**Comment by gvolpe** [comment]
Just experienced the same issue, such a bad UX!
